pulpcore/cli/container/context.py

```diff
@@ -48,9 +49,10 @@ class PulpContainerNamespaceContext(PulpEntityContext):
     HREF = "container_container_namespace_href"
     ID_PREFIX = "pulp_container_namespaces"
     CAPABILITIES = {"roles": [PluginRequirement("container", "2.11.0")]}
+    HREF_PATTERN = r"(?P<plugin>pulp_container)/(?P<resource_type>namespaces)/"
```

Comment:
This is more or less an artefact from https://github.com/pulp/pulp-cli/blob/70b6979c8a9e5bd89b3a3cb296b1580b5e9cf5f4/pulpcore/cli/common/context.py#L759. We still check for the resource type in `resource_option`. It is not currently used anywhere, but, I left those matching named groups untouched, similarly to pulp_ansible remotes.

Reply:
What happens if we do this?
```suggestion
    HREF_PATTERN = r"pulp_container/namespaces/"
```

Reply:
Nothing will change now. It is there for the sake of possible future use because we have got the `resource_option` option that reads the matched groups and initializes a corresponding resource (entity_ctx) class further on by relying on the data in those groups.

https://github.com/pulp/pulp-cli/blob/70b6979c8a9e5bd89b3a3cb296b1580b5e9cf5f4/pulpcore/cli/common/generic.py#L489-L490

Reply:
Well if anything, in any case, the plugin should be "container", and the resource_type is unknown or also "container".
But i see that the extraction there is at least resiliant to missing matches.